apps/wallet-mobile/src/legacy/DeveloperScreen.tsx

```diff
@@ -109,6 +112,14 @@ export const DeveloperScreen = () => {
           }}
         />
 
+        <Button
+          title={`Reset Analytics Consent (${isConsentRequested})`}
```

Comment:
intl?

Reply:
ah ok developer screen !

Reply:
That's the developer screen, no translations there.